packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -817,7 +818,7 @@ async function createReferencePriceFeedForFinancialContract(
 
 function getFinancialContractIdentifierAtAddress(web3, financialContractAddress) {
   try {
-    const ExpiringMultiParty = getTruffleContract("ExpiringMultiParty", web3, "1.2.0");
```

Comment:
Is it OK to just use default latest version from `hre.getContract`?

Reply:
I think so. But if not, we'll need to import a pinned version of core to get them.